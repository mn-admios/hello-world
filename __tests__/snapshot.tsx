import { render } from "@testing-library/react";
import Home from "@/pages/index";
import AboutPage from "@/pages/about/index";

it("renders homepage unchanged", () => {
  const { container } = render(<Home />);
  expect(container).toMatchSnapshot();
});

it("renders aboutpage unchanged", () => {
  const { container } = render(<AboutPage />);
  expect(container).toMatchSnapshot();
});
