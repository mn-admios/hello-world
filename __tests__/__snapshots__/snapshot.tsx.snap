// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders aboutpage unchanged 1`] = `
<div>
  <div>
    <h2>
      This is the about page
    </h2>
    <span>
      Go
    </span>
    <a
      href="/"
    >
      Back
    </a>
  </div>
</div>
`;

exports[`renders homepage unchanged 1`] = `
<div>
  <div>
    <h2
      data-testid="headingTestID"
    >
      Hello World Next App
    </h2>
    <span>
      Testing route, go to: 
    </span>
     
    <a
      href="/about"
    >
      About
    </a>
  </div>
</div>
`;
